### PR TITLE
Us03/fe202 multiple choice exercise

### PR DESCRIPTION
**Pull request by QUanlE79**
Multiple choice exercise type for learning feature
image

### CODE REVIEW

**Review by thaihiepnguyen — COMMENTED** [review]
Screenshot 2024-04-17 000619
Screenshot 2024-04-17 000654


I don't why I can not run it on my environment. 
Could you help me to resolve it?

Thank you.

**Comment by aduongdnai** [comment]
Please merge from Dev and resolve conflict

File: Assets/Scripts/Exercise/ExerciseMananger.cs [hunk 1]
```diff
@@ -32,17 +32,27 @@ public class ExerciseMananger : MonoBehaviour
...
+
     private void Awake()
     {
         
     }
     private async void Start()
     {
-        StartCoroutine(GetRequest(GlobalVariable.server_url + "/exercises", (response)=> { 
-        
+        StartCoroutine(GetRequest(GlobalVariable.server_url + "/exercises", (response) =>
```
[the remark this answers is not in the document]
Comment (QUanlE79) [reply Assets/Scripts/Exercise/ExerciseMananger.cs]: I have adjusted it

**Review by aduongdnai — APPROVED** [review]
It is ok now

**Pull request merged**